vision_datasets/dataset/base_dataset.py

```diff
@@ -1,6 +1,6 @@
 from abc import ABC, abstractmethod
 
-from .dataset_info import BaseDatasetInfo
+from ..common import BaseDatasetInfo
 
 
 class BaseDataset(ABC):
```

Comment:
irispyv2 adds `get_targets(index)` method to get only targets without image. might be convenient if we could include it to this dataset interface.